packages/flutter/lib/src/foundation/change_notifier.dart

```diff
@@ -106,11 +106,17 @@ class ChangeNotifier implements Listenable {
   List<VoidCallback?> _listeners = List<VoidCallback?>.filled(0, null);
   int _notificationCallStackDepth = 0;
   int _reentrantlyRemovedListeners = 0;
-  bool _debugDisposed = false;
+  bool _isDisposed = false;
```

Comment:
Given that this field isn't strictly for debugging (it's now indirectly exposed to developers), I've renamed it and it's associated assert function.